actions/lib/base.py

```diff
@@ -62,6 +62,9 @@ def _reset(self, org):
         else:
             self._client = Github(self.token, base_url=self.base_url)
 
+    def _temp_client(self, token, base_url):
+        return Github(token, token, base_url=base_url)
```

Comment:
`base_url` can be blank or `None` so we need to default it in those cases.

Reply:
OK.  I add the code.

Reply:
Need to also check for `''` (blank) and I think we should default to `DEFAULT_API_URL` if the config does not have the value

Reply:
OK. I add it.